dbbackup/management/commands/mediabackup.py

```diff
@@ -95,6 +90,11 @@ def get_source_dir(self):
 
     def create_backup_file(self, source_dir, backup_basename, **kwargs):
         temp_dir = tempfile.mkdtemp(dir=dbbackup_settings.TMP_DIR)
+        # Amazon S3 data model is a flat structure. However, Amazon Console
```

Comment:
Sorry, if I don't understand what you want to say. 
`/` is usable (mb because I use it). When I try to add backup filename `my_backup_folder/media.tar.gz` I get an error due to have no dir `my_backup_folder` in tmp folder (on unix system, mb it's not actual for win platform). The simplest way to avoid it - add this trick. Mb `os.makedirs` will be better approach?